src/components/CalendarGrid.vue

```diff
@@ -124,8 +123,7 @@ export default {
 				navLinks: true,
 				// Localization
 				...getDateFormattingConfig(),
-				locale: getFullCalendarLocale(getLocale(), this.locale),
-				firstDay: getFirstDayOfWeekFromMomentLocale(this.locale),
+				locale: getFullCalendarLocale(getLanguage(), this.locale),
```

Comment:
We need some clarity on locale vs language for both FullCalendar and Nextcloud and if the two definitions match. With this change the FC locale is set to the NC language. This feels wrong, but maybe it is correct.

Reply:
Good point, it is the core of the current issue. NC seperates the two, but in FC locale = language. FC doesn't have a seperate way to input a locale. That is why, in the GetFullCalendarLocale, I have replaced the content of the date and time fields with NC moment calls. Should I rename locale to localeAndLang maybe?